sys/include/net/gnrc/rpl.h

```diff
@@ -82,6 +82,40 @@
  *   CFLAGS += -DCONFIG_GNRC_RPL_WITHOUT_VALIDATION
  *   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  *
+ * - This RPL implementation currently only supports storing mode.
+ *   That means, in order to have downwards routes to all nodes the
+ *   storage space within [`gnrc_ipv6`'s Neighbor Information Base](@ref net_gnrc_ipv6_nib)
+ *   must be big enough to store information for each node.
+ *
+ *   For a random topology of n nodes, to ensure you can reach every node from the root,
+ *   set `CONFIG_GNRC_IPV6_NIB_NUMOF` == `CONFIG_GNRC_IPV6_NIB_OFFL_NUMOF` == n.
+ *
+ *   e.g. for n = 50 set
+ *   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ {.mk}
+ *   CFLAGS += -DCONFIG_GNRC_IPV6_NIB_NUMOF=50
+ *   CFLAGS += -DCONFIG_GNRC_IPV6_NIB_OFFL_NUMOF=50
+ *   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ *
+ * - If you want to allow for alternative parents, increase the number of
+ *   default routers in the NIB.
+ *
+ *   e.g. for one alternative parent, set
+ *   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ {.mk}
+ *   CFLAGS += -DCONFIG_GNRC_IPV6_NIB_DEFAULT_ROUTER_NUMOF=2
+ *   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ *
+ * TODO
+ * ------
+ *
+ * The GNRC RPL implementation only implements storing mode with OF0 ([RFC6552](https://tools.ietf.org/html/rfc6552)).
+ * The RPL routing header is parsed by the nodes, but anything else for non-storing mode is missing.
```

Comment:
Just noticed this while reading this again. The RPL routing header is only parsed with the `gnrc_rpl_srh` module compiled in.

```suggestion
 * The RPL routing header is parsed by the nodes when the
 * [@c gnrc_rpl_srh](@ref net_gnrc_rpl_srh) module is used, but anything else
 * for non-storing mode is missing.
```